AdminDashboard/Models/Database/DbAccount.cs

```diff
@@ -0,0 +1,12 @@
+﻿using AdminDashboard.Main.Enumerations;
+using System.Collections.Generic;
+
+namespace AdminDashboard.Models.Database
```

Comment:
The repository layer only deals with database objects, not domain models for separations of concern